versions/3.0.md

```diff
@@ -2599,7 +2602,7 @@ definitions:
       ]
     },
     "Cat": {
-      "description": "A representation of a cat",
+      "description": "A representation of a cat. Note that `Cat` will be used as the discriminator value.",
```

Comment:
Please add an example:

Below are examples showing an array of Pet objects using `petType` as a discriminator:
```yaml
- name: "Fluffy"
  petType: "Cat"
  huntingSkill: "lazy"
- name: "Rover"
  petType: "Dog"
  packSize: 6
```

```json
[
  {
    "name" : "Fluffy",
    "petType" : "Cat",
    "huntingSkill" : "lazy"
  },
  {
    "name" : "Rover",
    "petType" : "Dog",
    "packSize" : 6
  }
]

```